File: server-side/models/guess.js
const Sheet = require("./sheet");

class Guess extends Sheet {
    constructor(objectiveWord, drawing, playerName) {
        super(objectiveWord, playerName);
        this.drawing = drawing;
        
        this.answer = null;
    }

    setAnswer(answer) {
        this.answer = answer;
    }

    getAnswer() {
        return this.answer;
    }
}

module.exports = Guess;

File: client-side/core/src/com/sketchy/game/Models/Guess.java
package com.sketchy.game.Models;

public class Guess extends Sheet {

    private String answer;

    public Guess(String objectiveWord, String playerName, String answer) {
        super(objectiveWord, playerName);
        this.answer = answer;
    }

    public String getAnswer() {
        return answer;
    }

    public void setAnswer(String answer) {
        this.answer = answer;
    }
}
